src/main/java/ca/bc/gov/api/oracle/legacy/entity/ClientLocationPKEntity.java

```diff
@@ -0,0 +1,20 @@
+package ca.bc.gov.api.oracle.legacy.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.With;
+import org.springframework.data.relational.core.mapping.Column;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@With
+@Builder
+public class ClientLocationPKEntity {
```

Comment:
Can we please call it ClientLocationIdEntity for this type of classes?

Reply:
Done